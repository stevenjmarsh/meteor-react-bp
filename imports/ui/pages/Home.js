// @flow
import React from "react";
import { defaultPageStyle } from "../styles";

const Home = () => (
  <div className={defaultPageStyle.className}>
    <h1>Home</h1>
  </div>
);

export default Home;
