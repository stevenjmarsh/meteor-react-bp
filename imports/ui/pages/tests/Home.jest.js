/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
/* eslint-disable import/no-extraneous-dependencies */

import React from "react";
import renderer from "react-test-renderer";
import Home from "../Home";

describe("<Home />", function () {
  it("matches render snapshot", function () {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
