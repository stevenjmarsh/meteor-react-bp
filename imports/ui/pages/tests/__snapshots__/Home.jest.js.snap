exports[`<Home /> matches render snapshot 1`] = `
<div
  className="css-wlbfil">
  <h1>
    Home
  </h1>
</div>
`;
